### ISSUE TITLE
[ignore] Testing Github Actions (old cli version)

### ISSUE THREAD

**Issue by iowillhoit** [forcedotcom/cli#2216]
> **Note** 
> Before you submit your issue, make sure that:
> - You're using the latest version of Salesforce CLI.
> - You've searched both open and closed issues for related posts.
> - You've used the `doctor` command to diagnose common issues. 
> - You understand that GitHub Issues don't adhere to any agreement or SLA.
>   - If you require immediate assistance, use official channels such as Salesforce Customer Support.

### Summary

@salesforce/cli/1.60.5 darwin-arm64 node-v18.15.0 pluginVersions

### Steps To Reproduce

> **IMPORTANT**
> Provide a repository that's configured to reproduce the issue. If you are unable to provide a repo, please explain why not. The more info we have from the start, the faster we can resolve your issue.
> We may close your issue if you don't include proper instructions.
>
> - Generate a project with `sf project generate` or fork [dreamhouse-lwc](https://github.com/trailheadapps/dreamhouse-lwc).
> - Provide detailed step-by-step instructions on how to reproduce the issue.

### System Information

```json
PASTE_VERSION_OUTPUT_HERE
```

**Comment by iowillhoit** [comment]
```
{
  "cliVersion": "@salesforce/cli/1.81.6",
  "architecture": "darwin-arm64",
  "nodeVersion": "node-v18.15.0",
  "osVersion": "Darwin 22.5.0",
  "shell": "zsh",
  "rootPath": "/Users/ewillhoit/.nvm/versions/node/v18.15.0/lib/node_modules/@salesforce/cli",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 2.3.0 (core)",
    "@oclif/plugin-commands 2.2.15 (core)",
    "@oclif/plugin-help 5.2.9 (core)",
    "@oclif/plugin-not-found 2.3.24 (core)",
    "@oclif/plugin-plugins 3.1.2 (core)",
    "@oclif/plugin-search 0.0.17 (core)",
    "@oclif/plugin-update 3.1.16 (core)",
    "@oclif/plugin-version 1.3.4 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.37 (core)",
    "@oclif/plugin-which 2.2.21 (link) /Users/ewillhoit/dev/plugin-which",
    "@salesforce/cli 1.81.6 (core)",
    "apex 2.2.22 (core)",
    "auth 2.7.17 (core)",
    "custom-metadata 2.1.22 (link) /Users/ewillhoit/dev/plugin-custom-metadata",
    "data 2.3.20 (core)",
    "deploy-retrieve 1.10.4 (core)",
    "info 2.6.19 (link) /Users/ewillhoit/dev/plugin-info",
    "limits 2.3.17 (core)",
    "login 1.2.11 (core)",
    "org 2.9.3 (core)",
    "packaging 1.16.11 (user)",
    "schema 2.3.10 (core)",
    "settings 1.4.10 (core)",
    "sobject 0.1.22 (core)",
    "source 2.10.11 (core)",
    "telemetry 2.2.0 (core)",
    "templates 55.4.18 (core)",
    "trust 2.4.20 (core)",
    "user 2.3.14 (core)"
  ]
}
```

**Comment by iowillhoit** [comment]
Looks good. Suggested latest, validated when latest was provided 👍
